features/251-templates/index.md

```diff
@@ -175,3 +175,11 @@ Unchanged templates:
 
 Continue? ([y]/n)
 ```
+
+### `skycli app download-templates [-d <dir>] [-y]`
+
+It download the remote templates to `<dir>`. `<dir>` is `./templates` by default.
+
+If `<dir>` is non-empty, it shows a prompt to warn the developer that files may be overwritten.
```

Comment:
Yes.

Reply:
I think we should only prompt if it would overwrite some files. For example, developer may want use compiled HTML (e.g. `sed` API key placeholder) as template, and use `.gitignore` for output template HTML.

Reply:
OK. I will revise it.